content/word-lists/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Overview"
+---
+
+# Language recommendation lists 
+
+The Inclusive Naming Initiative produces 3 tiers of language recommendation. These are compiled into lists for easy consumption by adopters. Terms are separated into lists based on the severity of the term in question as well as the level of review the terms have received.
+
+## Tier 1: Replace Immediately
+
+Tier 1 terms should be replaced whenever encountered. 
```

Comment:
Do we want to specify the places where things need to change. For eg, IMO Tier 1 is words that need to be changing in CODE as well as writing, documentation, speech. Tier 2 is maybe don't add net new such terms to code, writing, docs, speech but can leave as is in existing code. Tier 3 is not avoid in writing and speech.

Reply:
I think really that's down to people to decide on an individual basis. We can't really make a blanket judgement on where things should be swapped out because it's based on individual circumstances for each project.

Reply:
@pritianka Agree with @Nytelife26 - I think that's down to people to decide on an individual basis.